src/main/python/run_regression.py

```diff
@@ -0,0 +1,176 @@
+#!/Users/peiliny/miniconda3/bin/python
+"""
+Anserini: An information retrieval toolkit built on Lucene
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+from __future__ import print_function
+import os
+import yaml
+import itertools
+from subprocess import call, Popen, PIPE
+import argparse
+
+OKBLUE = '\033[94m'
+FAIL = '\033[91m'
+ENDC = '\033[0m'
+
+def check_output(command):
```

Comment:
I think we were going to use PEP8 four-space indent?

Reply:
Same question...shouldn't we use four-space indent here?